test/test_case.rb

```diff
@@ -1,6 +1,6 @@
 require 'test-unit'
 require_relative '../lib/ontologies_api_client'
-require_relative '../config/config'
+require_relative '../config/config.test'
```

Comment:
you don't need this, locally you need to do ` cp config/config.test.rb config/config.rb` to run test. 
and in GitHub CI it is done automatically  https://github.com/ontoportal-lirmm/ontologies_api_ruby_client/blob/master/.github/workflows/ruby.yml#L33